code/datums/perks/job.dm

```diff
@@ -317,7 +331,7 @@
 
 /datum/perk/rezsickness/severe/fatal
 	name = "Fatal Revival Sickness"
-	desc = "You've recently died and have been brought back to life, the experience has left you utterly physically and mentally wrecked, you better find a bed to recover in..."
+	desc = "You have a (troubling) penchant for getting killed at this point. Experiencing multiple deaths and revivals has left you utterly physically and mentally wrecked, you better stay in bed for now, or you risk dying a fourth time, this time, for good."
```

Comment:
""You have a (troubling) penchant for getting killed at this point. Experiencing multiple deaths and revivals" This doesn't really make sense given revival sickness is entirely based on toughness, the tougher you are the lest severe it is.

Reply:
You're right, I swore fatal rez sickness happened when you died with the other debuffs still active, I'll change the three to better reflect the toughness stat influence